doc/Hacking.md

```diff
@@ -17,7 +17,12 @@ Anything printed by `logger.dbg` starts with `DEBUG`.
 04/06/17-21:44:53 DEBUG foo
 ```
 
-## Bug hunting in kpv
+In production code, remember that arguments are *always* evaluated in Lua, so,
+don't inline complex computations in logger functions' arguments.
+If you *really* have to, hide the whole thing behind a `DEBUG.is_on` branch,
```

Comment:
We've just been calling it `dbg` in newer code. Pretty much only the oldest still uses `DEBUG`.